testsuite/podman_runner/12_run_proxy.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+set -euxo pipefail
+
+create_proxy_configuration() {
+
+  export PROXY_UTILS=$HOME/proxy
+
+  sudo --login podman exec uyuni-server-all-in-one-test bash -c \
+    'cp /root/ssl-build/RHN-ORG-TRUSTED-SSL-CERT /root/ssl-build/uyuni-server-all-in-one-test/server.crt /root/ssl-build/uyuni-server-all-in-one-test/server.key /tmp'
+
+  mkdir --parents \
+    $PROXY_UTILS \
+    $PROXY_UTILS/proxy-squid-cache \
+    $PROXY_UTILS/proxy-rhn-cache \
+    $PROXY_UTILS/proxy-tftpboot
+
+  ssh-keygen -t rsa -b 4096 -C "salt@uyuni-server-all-in-one-test" -f /tmp/test-all-in-one/id_openssh_rsa -N ""
+
+  cat <<EOF > $PROXY_UTILS/config.yaml
+server: uyuni-server-all-in-one-test
+ca_crt: |
+$(sudo --login sed 's/^/  /' /tmp/test-all-in-one/RHN-ORG-TRUSTED-SSL-CERT)
+proxy_fqdn: proxy-httpd
+max_cache_size_mb: 2048
+server_version: 5.0.0 Beta1
```

Comment:
I'm not sure if the value this parameter it has some impact, or it's just informative.
But I would better parametrize this in a way that we don't need to commit a change here for every new version.

Reply:
You could do it similarly to what we do for the containerized server. See https://github.com/uyuni-project/uyuni/blob/master/testsuite/podman_runner/07_start_server.sh#L31